plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/custom-validation-handler.ts

```diff
@@ -21,17 +20,24 @@ const customValidationHandler = (
 
 	if (
 		field === 'postcode' &&
-		customValues.country &&
+		country &&
 		! isPostcode( {
 			postcode: inputObject.value,
-			country: customValues.country,
+			country,
 		} )
 	) {
 		inputObject.setCustomValidity(
 			__( 'Please enter a valid postcode', 'woocommerce' )
 		);
 		return false;
 	}
+
+	if ( field === 'email' && ! isEmail( inputObject.value ) ) {
+		inputObject.setCustomValidity(
+			__( 'Please enter a valid email address', 'woocommerce' )
+		);
+		return false;
+	}
 	return true;
```

Comment:
```suggestion
	if ( field === 'email' && ! isEmail( inputObject.value ) ) {
		inputObject.setCustomValidity(
			__( 'Please enter a valid email address', 'woocommerce' )
		);
		return false;
	}
	
	return true;
```

What are your thoughts on keeping the space above the final return for readability-reasons?